example/basic/README.md

```diff
@@ -1,14 +1,19 @@
 # Basic sample to configure and use Dynatrace OpenTelemetry exporter
 
 The sample will write a single value to `otel.dynatrace.com.golang` metric. By default, the sample
-will connect to the local Dynatrace metrics endpoint.
+will connect to the local OneAgent endpoint.
 
 To send metric directly to Dynatrace server metric ingest, set environment variables `ENDPOINT`and `API_TOKEN`.
 
 ```bash
 $ go build .
-$ ENDPOINT=https://<environment ID>.dev.dynatracelabs.com/api/v2/metrics/ingest API_TOKEN=<API Token> ./basic
+
+$ # Export to local OneAgent endpoint
+$ ./basic
 Exact
 otel.dynatrace.com.golang gauge,min=1.000000,max=1.000000,sum=1.000000,count=1
 Dynatrace returned: {"linesOk":1,"linesInvalid":0,"error":null}
```

Comment:
Seems like this debug output is no longer there although I think it would make sense. Please add it back or remove it from the readme as otherwise it's misleading.